extensions/ci/main.ts

```diff
@@ -0,0 +1,55 @@
+import { Package, PhylumApi } from "phylum";
+
+// Ensure required arguments are present.
+const args = Deno.args.slice(0);
+if (args.length < 4) {
+  console.error(
+    "Usage: phylum ci <PROJECT> [--group <GROUP>] <LABEL> <BASE> <LOCKFILE...>",
+  );
+  Deno.exit(1);
+}
+
+// Find optional groups argument.
+let group = undefined;
+const groupArgsIndex = args.indexOf("--group");
+if (groupArgsIndex != -1) {
+  const groupArgs = args.splice(groupArgsIndex, groupArgsIndex + 1);
```

Comment:
```suggestion
  const groupArgs = args.splice(groupArgsIndex, 2);
```
It would work as written when `--group` was the second argument (i.e., `groupArgsIndex == 1`)
This does not need to be changed in this review since this entire file is expected to be removed...just pointing it out in case it helps anyone else reading this.

Reply:
Yes...the position of `--group` is not fixed. This is actually how this bug was discovered. The inclusion of this extension in `phylum-ci` for testing was done such that the `phylum-ci` code called it as `phylum ci <PROJECT> <LABEL> [--group <GROUP>] <BASE> <LOCKFILE...>` (moved `<LABEL>` to come before `--group`).

The [docs for `.splice()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) show the syntax for a two argument call as `splice(start, deleteCount)`, where `deleteCount` means:

> An integer indicating the number of elements in the array to remove from `start`.

So, in the example above, `start` would be the value of `groupArgsIndex`, which is 2 and then `deleteCount` would be (2+1), which is 3 and one more than expected.

Reply:
Oh, duh.